Tests/Resources/app/config/cmf_routing.phpcr.yml

```diff
@@ -3,4 +3,4 @@ cmf_routing:
         persistence:
             phpcr:
                 enabled: true
-                route_basepath: /test/routing
+                route_basepaths: /test/routing
```

Comment:
not sure. it does not cost us much to convert string to array here for when you have only one path. @WouterJ what do you think is better DX? do we create confusion by allowing single value or is it convenient?